### PR TITLE
Update to ASP.net Configuration page

### PR DESCRIPTION
**Pull request by JohnSurina**
The Configuration documentation page does not explicitly describe how one can collect configuration values from the IConfiguration service (provided via dependency injection).
I would like to add a section to this documentation article explicitly describing how one can do that.

[Internal Preview URL](https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-6.0&branch=pr-en-us-23369#access-configuration-with-dependency-injection-di).

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=23369) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=23369) before we can accept your contribution.<br/><br/>:x: JohnSurina [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=23369)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetCore.Docs?pullRequest=23369) it.</sub>

File: aspnetcore/fundamentals/configuration/index.md [hunk 1]
```diff
@@ -1610,6 +1610,14 @@ The following code shows how to use the custom `EFConfigurationProvider` in *Pro
 
 <a name="acs"></a>
 
+## Generic access to configuration information
```
[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply aspnetcore/fundamentals/configuration/index.md:1613]: @serpent5  can you suggest better wording?

**Review by Rick-Anderson — COMMENTED** [review]
@serpent5  please review this PR

**Review by serpent5 — APPROVED** [review]

**Comment by serpent5** [comment]
Thanks @JohnSurina. 🥇

**Pull request merged**